cmd/generator/main.go

```diff
@@ -112,6 +112,23 @@ var generatorCmd = &cobra.Command{
 			plugins = append(plugins, releasePlugins...)
 		}
 
+		// Ensure mannally added plugin are still keeped in the database
+		manuallyAdded := []*model.Plugin{}
+		for _, ep := range existingPlugins {
+			found := false
+			for _, p := range plugins {
+				if p.DownloadURL == ep.DownloadURL {
+					found = true
+				}
+			}
+
+			if !found {
+				manuallyAdded = append(manuallyAdded, ep)
+			}
+		}
+
+		plugins = append(plugins, manuallyAdded...)
```

Comment:
I would really like to sort all plugins first by `id` and then by `version`. That way manually added plugins would not always be added to the end of the list. But that change would generate a huge diff. Thoughts?

Reply:
Is this the order of the user facing list? I think that ordering by ID might not be optimal, since IDs might not end up alphabetically ordered as expected (i.e. I would like to have plugin "D", before plugin "E", but if the ids are "d" and "com.a.e", "E" will be listed first). I would try a more complex idea:
1. Group plugins by ID
2. Order the plugin inside each group by version.
3. Take the plugin Name of the highest version plugin and make that a "representative or the group".
4. Order the groups by the representative Name alphabetically.

That way, if I change the plugin name from `My awesome plugin` to `Awesome plugin` both will be together, and in the order of the one of the highest version. It will slightly break the alphabetic order, but it feels better for me.

Reply:
No, this output does should not affect the user-facing list: it was intended just for hygiene in the generated file.

I'm fine with a one-time diff this time around, @hanzei, if it simplifies things going forward.

Reply:
@lieut-data @larkox I did add the sorting method. PTAL.